modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/portlet/configuration/icon/DeleteOrganizationPortletConfigurationIcon.java

```diff
@@ -76,7 +78,16 @@ public String getOnClick(
 
 			sb.append("', '");
 
-			sb.append(portletRequest.getAttribute("view.jsp-backURL"));
+			String backURL = String.valueOf(
+				portletRequest.getAttribute("view.jsp-backURL"));
+
+			if ((backURL == null) || backURL.isEmpty()) {
```

Comment:
Just FYI, we have a util to do this kind of checking on Strings: [Validator.java](https://github.com/liferay/liferay-portal/blob/master/portal-kernel/src/com/liferay/portal/kernel/util/Validator.java#L1068).